source/clear-linux/tutorials/wp-install/wp-install.rst

```diff
@@ -3,204 +3,150 @@
-   command:
+4. To rename the top-level WordPress directory to “blog,” enter the following command:
 
-   .. code-block:: console
+   .. code-block:: bash
 
       sudo mv wordpress blog
 
 5. To remove the downloaded tar file, enter the following command:
 
-   .. code-block:: console
+   .. code-block:: bash
 
       sudo rm latest.tar.gz
 
-Setting up WordPress
+Set up WordPress
 --------------------
 
-With the WordPress components loaded into the
-:file:`/var/www/html/blog` directory, we can set everything up.
+Recall that you created a database and user when you installed phpMyAdmin during :ref:`Creating a Clear Linux based web server <web-server-install>`. To continue with the WordPress setup, go to: http://<your_website>/blog/wp-admin/install.php.
 
-Instead of editing the :file:`wp_config.php` file manually, we are using the
-web-based configuration tool to setup the database name and user. We created
-both in the :ref:`web server tutorial <web-server-install>`. In your browser,
-go to: http://<your_website>/blog/wp-admin/install.php.
-
-Your screen should look like figure 1:
+The WordPress language option screen appears, as shown in figure 1. Select :guilabel:`English` and click :guilabel:`Continue`.
 
 .. _fig-wp-install-1:
 
 .. figure:: figures/wp-install-1.png
     :alt: WordPress language selection
     :width:     600
 
-    WordPress language selection screen.
-
-Select :guilabel:`English` for the language and click the
-:guilabel:`Continue` button. :ref:`As shown on figure 1<fig-wp-install-1>`
+    Figure 1: WordPress language selection screen.
 
 
-
-The WordPress installation continues until the welcome screen shown in figure
-2 appears:
+The WordPress installation continues until the Welcome screen appears, as shown in figure 2:
 
 .. figure:: figures/wp-install-2.png
     :alt: WordPress welcome screen
     :width:     600
 
-    WordPress welcome screen.
-
-Click the :guilabel:`Let’s go!` button to enter the information.
+    Figure 2: WordPress welcome screen.
 
-Enter the database name, username, and password we used when creating the
-database:
+Click :guilabel:`Let’s go!`. Enter pertinent information in the screen that appears as shown in Figure 3 below.
 
    Database name:       WordPress
 
    Database username:   wordpressuser
 
    Database password:   wp-example
 
-Enter the following values for the database host and the table prefix:
-
    Database host:  localhost
 
    Table prefix:   wp\_
 
-Figure 3 shows the filled out fields.
+Figure 3 shows the completed fields:
 
 .. figure:: figures/wp-install-3.png
     :alt: Database connection details
     :width:     600
 
-    These details are needed for WordPress to connect to the database.
-
-.. note::
+    Figure 3: Information necessary for WordPress to connect to the database.
 
-   When you enter your password into the password field, it will be in clear
-   text and not asterisks.
+Click :guilabel:`Submit` to complete the setup.
```

Comment:
Revise to numbered step please. 
#. Click :guilabel:`Submit` to complete the setup.

Reply:
done